tests/db_migrator_test.py

```diff
@@ -247,3 +247,33 @@ def test_init_config_feature_migration(self):
         assert not diff
 
         assert not expected_db.cfgdb.get_table('CONTAINER_FEATURE')
+
+
+class TestLacpKeyMigrator(object):
+    def test_init_config_feature_migration(self):
```

Comment:
do we really need this function in class TestLacpKeyMigrator

Reply:
Fixed